### DISCUSSION TITLE
🤔 [QUESTION] Treemap 如何设置 radius

### DISCUSSION THREAD

**Discussion by Sunshine168** [antvis/G2Plot#3557] [Q&A] [answered]
RT , 在 G2 是可以这样设置的，但是在这边要怎么传参数
<img width="1610" alt="image" src="https://github.com/antvis/G2Plot/assets/15126380/0e632932-08ed-4f6d-9237-bca6f61e3fea">

**Comment by pearmini** [comment, marked as answer]
目前 G2Plot 是基于 G2 4.0 的，而 4.0 的 G2 用 polygon geometry 画的矩阵树图，polygon geometry 不支持圆角。

**Comment by Sunshine168** [comment]
好的， BTW 短期内 G2Plot 有计划升级到 v5 的 G2 么

**Comment by pearmini** [comment]
短期内还没有，有需要可以直接试试 v5 🤣
